lib/presentation/screens/favorite_screen/favorite_layout.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:spotify_prototype/presentation/screens/favorite_screen/bloc/favorite_states.dart';
+import 'package:spotify_prototype/presentation/screens/favorite_screen/bloc/favourite_bloc.dart';
+import 'package:spotify_prototype/presentation/widgets/liked_track_widget.dart';
+import 'package:spotify_prototype/presentation/widgets/refresh_widget.dart';
+import 'bloc/favorite_events.dart';
+
+class FavoriteLayout extends StatefulWidget {
+  const FavoriteLayout({Key? key}) : super(key: key);
+
+  @override
+  State<FavoriteLayout> createState() => _FavoriteLayoutState();
+}
+
+class _FavoriteLayoutState extends State<FavoriteLayout> {
+  @override
+  void initState() {
+    context.read<FavoriteBloc>().add(LoadFavoriteEvent());
+    super.initState();
+  }
+
+  Future loadList() async {
+    await Future.delayed(const Duration(milliseconds: 400));
```

Comment:
why the fuck is that here?

Reply:
It is a debouncing

Reply:
no, it's not a debouncing, I deleted the "await Future.delayed(const Duration(milliseconds: 400));"